.github/CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ Angus Lowe
   in `_device.py`.
   [#374](https://github.com/XanaduAI/pennylane/pull/374)
 
+* Default qubit state vector preparation operations can now be applied to subsets of qubits, and
```

Comment:
This might confuse people. What's a default qubit state vector preparation?
Suggestion "State vector preparation operations in the ``default.qubit`` plugin can now be ..."

Reply:
Okay this has been updated

Reply:
Thanks!